### PR TITLE
Process chapter edit

### PR DESCRIPTION
**Pull request by sapegin**

### CODE REVIEW

File: manuscript/infrastructure/01-processes.md [hunk 1]
```diff
@@ -11,57 +11,53 @@ When you think about processes, you get at least the following questions:
...
 
 ## How to Design a Development Process
 
-An important part of development practices is designing a development process. Especially early on this can be ad hoc as you don't have to worry about team productivity. Once more people become involved, you have to make sure they can contribute in a meaningful way.
-
-This is where managing issue tracking and pull requests well becomes vital. You should design a system for issues where you set them in a particular way. For example, each issue could gain a priority (0-4 for example), categorization (bug, feature request), and difficulty (0-3 for example). The labels you need depend on the type of the project and there's no one right answer on which labels to use.
-
-Most importantly it's a chance for you to apply reviews on code. You can discuss the implementation until it's in a satisfactory state. It's also a chance to grow new contributors to your project. They may even become maintainers one day and assume your position in the project.
+When you’re working on a project alone, ??????
```
Comment (bebraw) [inline manuscript/infrastructure/01-processes.md]: Maybe separate subsections for alone + team + alone with community?

Comment (bebraw) [reply manuscript/infrastructure/01-processes.md]: There's some progression in going from single person to many. If we do the chapter well, we'll explore different directions. Bigger projects tend to evolve loose team structures.

**Comment by sapegin** [comment]
I’ve finished the sentence but still not sure if we have to split it now.

**Comment by bebraw** [comment]
Ok, let's roll with this.

**Pull request merged**